packages/xo-web/src/common/bulk-icons.css

```diff
@@ -0,0 +1,3 @@
+.bulkIconsScale {
+  transform: scale(0.8);
```

Comment:
Why?

Reply:
I created a new CSS file because `Tooltip` doesn't work properly when children have the `style` attribute set. (I do not know why).